src/Dropdown.js

```diff
@@ -88,14 +97,18 @@ class Dropdown extends React.Component {
     return this.containerRef.current;
   }
 
+  getMenu() {
+    return this.menuRef.current;
+  }
+
   getMenuCtrl() {
     if (this._$menuCtrl) return this._$menuCtrl;
     this._$menuCtrl = this.getContainer().querySelector('[aria-expanded]');
     return this._$menuCtrl;
   }
 
   getMenuItems() {
-    return [].slice.call(this.getContainer().querySelectorAll('[role="menuitem"]'));
+    return [].slice.call(this.getMenu().querySelectorAll('[role="menuitem"]'));
```

Comment:
```suggestion
    const menuContainer = this.getMenu() || this.getContainer();
    return [].slice.call(menuContainer.querySelectorAll('[role="menuitem"]'));
```

Tests are failing here, but it's working in the docs site.  Seems like an issue with timing in the test environment.  Normally, the dropdown is supposed to toggle, then render the popper element (which assigns the menu ref), then trigger the focus code. In the tests, this menu ref is not yet assigned...

Anyways, I think we can safely add the old container as a fallback here just to satisfy the tests.  I don't see this being triggered when testing on the docs site.  This seems hacky, but it does follow whatever existing behavior was there previously anyway.

Reply:
Thanks again for the review! I pushed your fix and added a comment explaining why we do it. I agree it seems hacky, but I don't think it should ever break in real-life usage.